ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -476,6 +481,62 @@ ORK_CLASS_AVAILABLE
 
 
 /**
+ The `ORKTextScaleAnswerFormat `class represents an answer format that includes a slider control with text choices.
+ 
+ The scale answer format produces an `ORKScaleQuestionResult` object that contains a number whose value is the selected slider value.
+ */
+ORK_CLASS_AVAILABLE
+@interface ORKTextScaleAnswerFormat : ORKAnswerFormat
+
+/**
+ Returns an initialized text scale answer format using the specified values.
+ 
+ This method is the designated initializer.
+ 
+ @param textChoices                 Array of ORKTextChoice objects.
+ @param defaultValue                The default index of the scale. If this value is out of range, the slider is displayed without a default value.
+ @param vertical                    Pass `YES` to use a vertical scale; for the default horizontal scale, pass `NO`.
+ 
+ @return An initialized text scale answer format.
+ */
+- (instancetype)initWithTextChoices:(NSArray<ORKTextChoice *> *)textChoices
+                       defaultIndex:(NSInteger)defaultIndex
+                           vertical:(BOOL)vertical NS_DESIGNATED_INITIALIZER;
+
+/**
+ Returns an initialized text scale answer format using the specified values.
+ 
+ This method is a convenience initializer.
+ 
+ @param textChoices                 Array of ORKTextChoice objects.
+ @param defaultValue                The default index of the scale. If this value is out of range, the slider is displayed without a default value.
+ 
+ @return An initialized text scale answer format.
+ */
+- (instancetype)initWithTextChoices:(NSArray<ORKTextChoice *> *)textChoices
+                       defaultIndex:(NSInteger)defaultIndex;
+
+/**
+ An array of ORKTextChoices that represent the options to display in the slider. (read-only)
```

Comment:
Ill leave the `(read-only)` for now to stay consistent with other documentation. I'll follow up with a separate PR to remove them all.